### PR TITLE
georgia exemptions

### LINKED ISSUE #2390: Georgia Exemption
Thanks for filing an issue! Please remove any top-level headers that do not apply.

# Variable to add: name

# Variable to amend: name

# Parameter to add: name

# Parameter to amend: name

### PR DESCRIPTION
**Pull request by Eleanor-Zhu**
fixes #2390

### Summary
:memo::moneybag::test_tube:

This pull request adds new parameters, variables, and tests for the Georgia state income tax exemptions and credit for qualified child and dependent care expenses. It also adds a `changelog_entry.yaml` file to store the changelog entries for each pull request. These changes aim to improve the accuracy and completeness of the policy engine model for Georgia.

> _`Georgia` tax rules_
> _New files for parameters_
> _Autumn leaves `changelog`_

### Walkthrough
*  Add Georgia state income tax exemptions for personal and dependent filers ([link](https://github.com/PolicyEngine/policyengine-us/pull/2394/files?diff=unified&w=0#diff-d7b17dd76afa5bfe38e1cd63ca091b350f2e458bbd6e2eebc218f03a6e3eaa66R1-R12), [link](https://github.com/PolicyEngine/policyengine-us/pull/2394/files?diff=unified&w=0#diff-3b21c298a02f2635be21d25bb99915559707cab4be11f49ebd9a546f55b56d68L1-R21), [link](https://github.com/PolicyEngine/policyengine-us/pull/2394/files?diff=unified&w=0#diff-74a4cf1d3e54ae2b9dfbe692941003d24a2ebef9931df18254b0205cc3a7e0fbR1-R35))
* Add Georgia state income tax credit for qualified child and dependent care expenses ([link](https://github.com/PolicyEngine/policyengine-us/pull/2394/files?diff=unified&w=0#diff-41d72ed1bbfbbe3334a446814c9cf792924b70378da3755abf5313597b90a76fR1-R13))
* Add empty YAML file `changelog_entry.yaml` to the root directory for storing changelog entries ([link](https://github.com/PolicyEngine/policyengine-us/pull/2394/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+description: Amount to phase georgia dependent out by each person
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:1]:
```suggestion
description: Georgia phases the exemption amount out by this amount.
```

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+values:
+  2003-01-01: 3_000  
+metadata:
+  period: year
+  unit: currency-USD
+  label: "Georgia dependent phase out amount"
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:7]:
```suggestion
  label: Georgia dependent phase out amount
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:7]: and in other yaml files

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:1]: similar description style in other yaml files

File: policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+import numpy as np
+
+
+class ga_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "GA Exemptions"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py:8]:
```suggestion
    label = "Georgia Exemptions"
```

**Review by PavelMakarchuk — COMMENTED** [review]
format comments

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+description: Amount to phase georgia exemption out by each person.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml:1]:
```suggestion
description: Georgia phases-out the exemptions out at this rate depending on the filing status.
```

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  label: Georgia dependent phase out amount
+  
+  reference:
+  - title: 2022 Form 500 Georgia Income Tax Return
+    href: https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500
+  - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26
+    href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=2c053fd5-32c1-4cc1-86b0-36aaade9da5b&pdistocdocslideraccess=true&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3JT-00008-00&pdcomponentid=234187&pdtocnodeidentifier=ABWAALAADAAL&ecomp=k2vckkk&prid=4862391c-e031-443f-ad52-ae86c6bb5ce2
+
+JOINT:
+  1986-01-01: 3_700
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml:14]: To match the law and avoid needing to multiply in the formula
```suggestion
  1986-01-01: 7_400
```

Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml:14]: 1986 refers to the federal IRC. It looks like the law was most recently amended effective 2020 per the notes at the bottom. Same for other breakdowns and parameters.
```suggestion
  2020-01-01: 3_700
```

File: policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ga.tax.income.exemptions
+        filing_status = tax_unit("filing_status", period)
+
+        # Personal Exemptions
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        num_exemptions = tax_unit.sum(head + spouse)
+        personal_exemptions = num_exemptions * p.personal[filing_status]
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py:26]: Delete prior 4 lines
```suggestion
        personal_exemptions = p.personal[filing_status]
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        "https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500",
+        "https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=2c053fd5-32c1-4cc1-86b0-36aaade9da5b&pdistocdocslideraccess=true&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A6348-G0H1-DYB7-W3JT-00008-00&pdcomponentid=234187&pdtocnodeidentifier=ABWAALAADAAL&ecomp=k2vckkk&prid=4862391c-e031-443f-ad52-ae86c6bb5ce2",
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ga.tax.income.exemptions
+        filing_status = tax_unit("filing_status", period)
+
+        # Personal Exemptions
+        person = tax_unit.members
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ga/tax/income/exemptions/ga_exemptions.py:22]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml [hunk 8]
```diff
@@ -0,0 +1,22 @@
+description: Georgia phases-out the exemptions out at this amount depending on the filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Georgia dependent phase out amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml:5]:
```suggestion
  label: Georgia dependent exemption amount
```

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml [hunk 9]
```diff
@@ -0,0 +1,22 @@
+description: Georgia phases-out the exemptions out at this amount depending on the filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/personal.yaml:1]:
```suggestion
description: Georgia provides the following exemptions amount depending on the filing status.
```

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+description: Georgia provides the following exemption amount for each dependent.
+values:
+  2003-01-01: 3_000  
+metadata:
+  period: year
+  unit: currency-USD
+  label: Georgia dependent exemption amount
+  reference:
+  - title: 2022 Form 500 Georgia Income Tax Return
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:9]: Add a page and line number reference. If possible could you find a different version of this that's searchable, e.g. a pdf? Same for the other reference.

File: policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+    href: https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500
+  - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml:11]: Same for other
```suggestion
  - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7-26(b)(3)
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**